financialaid/serializers.py

```diff
@@ -133,31 +128,22 @@ def save(self):
         """
         Save method for this serializer
         """
-        tier_program = self.validated_data["tier_program"]
+        # Save FinancialAid instance
         self.instance.status = self.validated_data["action"]
-        email_data = {
-            "acting_user": self.context["request"].user,
-            "financial_aid": self.instance
-        }
         if self.instance.status == FinancialAidStatus.APPROVED:
-            self.instance.tier_program = tier_program
-            email_data.update({
-                "subject": FINANCIAL_AID_APPROVAL_SUBJECT_TEXT,
-                "body": FINANCIAL_AID_APPROVAL_MESSAGE_BODY
-            })
+            self.instance.tier_program = self.validated_data["tier_program"]
         elif self.instance.status == FinancialAidStatus.REJECTED:
+            # Not currently a valid status to save as
             self.instance.tier_program = get_no_discount_tier_program(self.instance.tier_program.program_id)
-            email_data.update({
-                "subject": FINANCIAL_AID_REJECTION_SUBJECT_TEXT,
-                "body": FINANCIAL_AID_REJECTION_MESSAGE_BODY
-            })
         elif self.instance.status == FinancialAidStatus.PENDING_MANUAL_APPROVAL:
-            email_data.update({
-                "subject": FINANCIAL_AID_DOCUMENTS_SUBJECT_TEXT,
-                "body": FINANCIAL_AID_DOCUMENTS_MESSAGE_BODY
-            })
+            pass
```

Comment:
Can we remove this `elif` block since it's not doing anything? If you want to say that `PENDING_MANUAL_APPROVAL` is intentionally left blank, leave a comment instead